addons/website/models/website.py

```diff
@@ -288,9 +288,19 @@ def configurator_skip(self):
     @api.model
     def configurator_apply(self, **kwargs):
         def set_colors(selected_palette):
-            url = '/website/static/src/scss/options/user_values.scss'
-            custo = {'color-palettes-name': "'%s'" % selected_palette}
-            self.env['web_editor.assets'].make_scss_customization(url, custo)
+            if selected_palette.get('is_recommended'):
+                url = '/website/static/src/scss/options/colors/user_color_palette.scss'
+                values = {}
+                for k, v in selected_palette.items():
+                    color_match = re.match('^color([1-5])$', k)
+                    if color_match:
+                        color_key = 'o-color-' + color_match.group(1)
+                        values[color_key] = v
```

Comment:
Mmh, why not receiving the correct data structure directly instead of having to parse it? Something like
```
selected_palette: {
    'is_recommended': true/false,
    'colors': [a, b, c, d, e]
}
```

In fact we could even have no need of a structure: selected_palette = the name of the chosen palette or a list of colors.